--- /usr/local/lib/python3.5/dist-packages/torch/nn/modules/container.py
+++ /usr/local/lib/python3.5/dist-packages/torch/nn/modules/container.py
@@ -1,31 +1,11 @@
 class Sequential(Module):
     r"""A sequential container.
-    Modules will be added to it in the order they are passed in the
-    constructor. Alternatively, an ``OrderedDict`` of modules can be
-    passed in. The ``forward()`` method of ``Sequential`` accepts any
-    input and forwards it to the first module it contains. It then
-    "chains" outputs to inputs sequentially for each subsequent module,
-    finally returning the output of the last module.
+    Modules will be added to it in the order they are passed in the constructor.
+    Alternatively, an ordered dict of modules can also be passed in.
 
-    The value a ``Sequential`` provides over manually calling a sequence
-    of modules is that it allows treating the whole container as a
-    single module, such that performing a transformation on the
-    ``Sequential`` applies to each of the modules it stores (which are
-    each a registered submodule of the ``Sequential``).
+    To make it easier to understand, here is a small example::
 
-    What's the difference between a ``Sequential`` and a
-    :class:`torch.nn.ModuleList`? A ``ModuleList`` is exactly what it
-    sounds like--a list for storing ``Module`` s! On the other hand,
-    the layers in a ``Sequential`` are connected in a cascading way.
-
-    Example::
-
-        # Using Sequential to create a small model. When `model` is run,
-        # input will first be passed to `Conv2d(1,20,5)`. The output of
-        # `Conv2d(1,20,5)` will be used as the input to the first
-        # `ReLU`; the output of the first `ReLU` will become the input
-        # for `Conv2d(20,64,5)`. Finally, the output of
-        # `Conv2d(20,64,5)` will be used as input to the second `ReLU`
+        # Example of using Sequential
         model = nn.Sequential(
                   nn.Conv2d(1,20,5),
                   nn.ReLU(),
@@ -33,8 +13,7 @@
                   nn.ReLU()
                 )
 
-        # Using Sequential with OrderedDict. This is functionally the
-        # same as the above code
+        # Example of using Sequential with OrderedDict
         model = nn.Sequential(OrderedDict([
                   ('conv1', nn.Conv2d(1,20,5)),
                   ('relu1', nn.ReLU()),
@@ -42,16 +21,6 @@
                   ('relu2', nn.ReLU())
                 ]))
     """
-
-    _modules: Dict[str, Module]  # type: ignore[assignment]
-
-    @overload
-    def __init__(self, *args: Module) -> None:
-        ...
-
-    @overload
-    def __init__(self, arg: 'OrderedDict[str, Module]') -> None:
-        ...
 
     def __init__(self, *args):
         super(Sequential, self).__init__()
@@ -62,7 +31,7 @@
             for idx, module in enumerate(args):
                 self.add_module(str(idx), module)
 
-    def _get_item_by_idx(self, iterator, idx) -> T:
+    def _get_item_by_idx(self, iterator, idx):
         """Get the idx-th item of the iterator"""
         size = len(self)
         idx = operator.index(idx)
@@ -71,18 +40,17 @@
         idx %= size
         return next(islice(iterator, idx, None))
 
-    @_copy_to_script_wrapper
-    def __getitem__(self, idx) -> Union['Sequential', T]:
+    def __getitem__(self, idx):
         if isinstance(idx, slice):
             return self.__class__(OrderedDict(list(self._modules.items())[idx]))
         else:
             return self._get_item_by_idx(self._modules.values(), idx)
 
-    def __setitem__(self, idx: int, module: Module) -> None:
-        key: str = self._get_item_by_idx(self._modules.keys(), idx)
+    def __setitem__(self, idx, module):
+        key = self._get_item_by_idx(self._modules.keys(), idx)
         return setattr(self, key, module)
 
-    def __delitem__(self, idx: Union[slice, int]) -> None:
+    def __delitem__(self, idx):
         if isinstance(idx, slice):
             for key in list(self._modules.keys())[idx]:
                 delattr(self, key)
@@ -90,26 +58,16 @@
             key = self._get_item_by_idx(self._modules.keys(), idx)
             delattr(self, key)
 
-    @_copy_to_script_wrapper
-    def __len__(self) -> int:
+    def __len__(self):
         return len(self._modules)
 
-    @_copy_to_script_wrapper
     def __dir__(self):
         keys = super(Sequential, self).__dir__()
         keys = [key for key in keys if not key.isdigit()]
         return keys
 
-    @_copy_to_script_wrapper
-    def __iter__(self) -> Iterator[Module]:
-        return iter(self._modules.values())
-
-    # NB: We can't really type check this function as the type of input
-    # may change dynamically (as is tested in
-    # TestScript.test_sequential_intermediary_types).  Cannot annotate
-    # with Any as TorchScript expects a more precise type
     def forward(self, input):
-        for module in self:
+        for module in self._modules.values():
             input = module(input)
         return input
 